modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -201,17 +201,18 @@ def partitioned_file(
 
         Parameters
         ----------
-        f : file to be partitioned
-        num_partitions : int, optional
+        f : file-like object
+            File handle of file to be partitioned.
+        num_partitions : int, default: None
             For what number of partitions split a file.
-            If not specified grabs the value from `modin.config.NPartitions.get()`
-        nrows : int, optional
+            If not specified grabs the value from `modin.config.NPartitions.get()`.
+        nrows : int, default: None
             Number of rows of file to read.
-        skiprows : array or callable, optional
+        skiprows : array or callable, default: None
```

Comment:
same here, I suggest using `optional` for these. In most cases I would prefer using `optional` when default value is `None`

Reply:
Fixed.